File: number-of-students-doing-homework-at-a-given-time.kt
class Solution {
  fun busyStudent(startTime: IntArray, endTime: IntArray, queryTime: Int): Int {
    var count = 0
    for (i in 0..startTime.size - 1) {
      if (startTime[i] <= queryTime && endTime[i] >= queryTime) count++
    }
    return count
  }
}

File: number-of-students-doing-homework-at-a-given-time.py
class Solution:
    def busyStudent(self, startTime: List[int], endTime: List[int], queryTime: int) -> int:
        def l(times: List[List[int]]) -> bool:
            startTime, endTime = times
            return True if queryTime >= startTime and queryTime <= endTime else False

        return len(list(filter(l, [[startTime[i], endTime[i]] for i in range(len(startTime))])))
